<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CCalibFromPlanesConfig</class>
 <widget class="QWidget" name="CCalibFromPlanesConfig">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>382</width>
    <height>705</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="maximumSize">
   <size>
    <width>382</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <item>
    <layout class="QFormLayout" name="params_group_form">
     <item row="0" column="0">
      <widget class="QLabel" name="downsample_factor_label">
       <property name="text">
        <string>Downsampling factor:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QSpinBox" name="downsample_factor_sbox">
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="value">
        <number>10</number>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="segment_label">
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-style:italic;&quot;&gt;Plane Segmentation&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="textFormat">
        <enum>Qt::RichText</enum>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <spacer name="label_spacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="normal_estimation_method_label">
       <property name="text">
        <string>Normal Estimation Method:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QComboBox" name="ne_method_cbox">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <item>
        <property name="text">
         <string>COVARIANCE_MATRIX</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>AVERAGE_3D_GRADIENT</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>AVERAGE_DEPTH_CHANGE</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="depth_smoothing_label">
       <property name="text">
        <string>Depth Dependent Smoothing:</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QCheckBox" name="depth_dependent_smoothing_check">
       <property name="text">
        <string/>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
       <property name="tristate">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="depth_change_threshold_label">
       <property name="text">
        <string>Maximum Depth Change Factor:</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QDoubleSpinBox" name="max_depth_change_factor_sbox">
       <property name="singleStep">
        <double>0.010000000000000</double>
       </property>
       <property name="value">
        <double>0.020000000000000</double>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="normal_smoothing_size_label">
       <property name="text">
        <string>Normal Smoothing Size:</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QDoubleSpinBox" name="normal_smoothing_size_sbox">
       <property name="singleStep">
        <double>0.500000000000000</double>
       </property>
       <property name="value">
        <double>10.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="7" column="0">
      <widget class="QLabel" name="angle_threshold_label">
       <property name="text">
        <string>Angle Threshold:</string>
       </property>
      </widget>
     </item>
     <item row="7" column="1">
      <widget class="QDoubleSpinBox" name="angle_threshold_sbox">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="value">
        <double>4.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="8" column="0">
      <widget class="QLabel" name="minimum_inliers_label">
       <property name="text">
        <string>Minimum Inliers Rate:</string>
       </property>
      </widget>
     </item>
     <item row="8" column="1">
      <widget class="QDoubleSpinBox" name="minimum_threshold_sbox">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="decimals">
        <number>3</number>
       </property>
       <property name="singleStep">
        <double>0.010000000000000</double>
       </property>
       <property name="value">
        <double>0.001000000000000</double>
       </property>
      </widget>
     </item>
     <item row="9" column="0">
      <widget class="QLabel" name="distance_threshold_label">
       <property name="text">
        <string>Distance Threshold:</string>
       </property>
      </widget>
     </item>
     <item row="9" column="1">
      <widget class="QDoubleSpinBox" name="distance_threshold_sbox">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="singleStep">
        <double>0.010000000000000</double>
       </property>
       <property name="value">
        <double>0.050000000000000</double>
       </property>
      </widget>
     </item>
     <item row="10" column="0">
      <widget class="QLabel" name="max_curvature_label">
       <property name="text">
        <string>Maximum Curvature:</string>
       </property>
      </widget>
     </item>
     <item row="10" column="1">
      <widget class="QDoubleSpinBox" name="max_curvature_sbox">
       <property name="singleStep">
        <double>0.100000000000000</double>
       </property>
       <property name="value">
        <double>0.100000000000000</double>
       </property>
      </widget>
     </item>
     <item row="11" column="0">
      <widget class="QLabel" name="max_cos_normal_sbox_2">
       <property name="text">
        <string>Max Planes Cos Normal:</string>
       </property>
      </widget>
     </item>
     <item row="11" column="1">
      <widget class="QDoubleSpinBox" name="max_cos_normal_sbox">
       <property name="decimals">
        <number>3</number>
       </property>
       <property name="maximum">
        <double>1.000000000000000</double>
       </property>
       <property name="singleStep">
        <double>0.100000000000000</double>
       </property>
       <property name="value">
        <double>0.998000000000000</double>
       </property>
      </widget>
     </item>
     <item row="12" column="0">
      <widget class="QLabel" name="dist_centre_plane_label">
       <property name="text">
        <string>Planes Centre Threshold:</string>
       </property>
      </widget>
     </item>
     <item row="12" column="1">
      <widget class="QDoubleSpinBox" name="dist_centre_plane_sbox">
       <property name="singleStep">
        <double>0.100000000000000</double>
       </property>
       <property name="value">
        <double>0.100000000000000</double>
       </property>
      </widget>
     </item>
     <item row="13" column="0">
      <widget class="QLabel" name="proximity_threshold_label">
       <property name="text">
        <string>Proximity Threshold:</string>
       </property>
      </widget>
     </item>
     <item row="13" column="1">
      <widget class="QDoubleSpinBox" name="proximity_threshold_sbox">
       <property name="decimals">
        <number>2</number>
       </property>
       <property name="singleStep">
        <double>0.100000000000000</double>
       </property>
       <property name="value">
        <double>0.400000000000000</double>
       </property>
      </widget>
     </item>
     <item row="14" column="0">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="solver_label">
         <property name="text">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-style:italic;&quot;&gt;Solver&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
         <property name="textFormat">
          <enum>Qt::RichText</enum>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="14" column="1">
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="15" column="0">
      <widget class="QLabel" name="max_iters_label">
       <property name="text">
        <string>Maximum Iterations:</string>
       </property>
      </widget>
     </item>
     <item row="15" column="1">
      <widget class="QSpinBox" name="max_iters_sbox">
       <property name="maximum">
        <number>1000</number>
       </property>
       <property name="value">
        <number>10</number>
       </property>
      </widget>
     </item>
     <item row="16" column="0">
      <widget class="QLabel" name="minimum_update_label">
       <property name="text">
        <string>Minimum update:</string>
       </property>
      </widget>
     </item>
     <item row="16" column="1">
      <widget class="QDoubleSpinBox" name="min_update_sbox">
       <property name="decimals">
        <number>5</number>
       </property>
       <property name="singleStep">
        <double>0.000100000000000</double>
       </property>
       <property name="value">
        <double>0.000010000000000</double>
       </property>
      </widget>
     </item>
     <item row="17" column="0">
      <widget class="QLabel" name="converge_error_label">
       <property name="text">
        <string>Convergence Error:</string>
       </property>
      </widget>
     </item>
     <item row="17" column="1">
      <widget class="QDoubleSpinBox" name="converge_error_sbox">
       <property name="decimals">
        <number>5</number>
       </property>
       <property name="singleStep">
        <double>0.000100000000000</double>
       </property>
       <property name="value">
        <double>0.000010000000000</double>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="button_group">
     <item>
      <widget class="QPushButton" name="extract_planes_button">
       <property name="text">
        <string>Extract Planes</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="match_planes_button">
       <property name="text">
        <string>Match Planes</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="calib_button">
       <property name="text">
        <string>Calibrate</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QPushButton" name="save_calib_button">
     <property name="text">
      <string>Save Calibration</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="save_params_button">
     <property name="text">
      <string>Save Parameters</string>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="spacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>ne_method_cbox</tabstop>
  <tabstop>depth_dependent_smoothing_check</tabstop>
  <tabstop>max_depth_change_factor_sbox</tabstop>
  <tabstop>normal_smoothing_size_sbox</tabstop>
  <tabstop>angle_threshold_sbox</tabstop>
  <tabstop>minimum_threshold_sbox</tabstop>
  <tabstop>distance_threshold_sbox</tabstop>
  <tabstop>extract_planes_button</tabstop>
  <tabstop>calib_button</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
